app/services/unsubscribe_service.rb

```diff
@@ -19,7 +19,7 @@ def unsubscribe!(subscriber, subscriptions, reason)
         end
 
         if no_other_subscriptions?(subscriber, subscriptions)
```

Comment:
might want to just put the condition in the if rather than two separate layers?

Reply:
Sure, amended the commit with that.